README.md

```diff
@@ -34,8 +33,7 @@ vars:
     intercom_schema: your_schema_name
 ```
 
-This package includes all source columns defined in the macros folder. To add additional columns to this package, do so using our pass-through column variables. This is extremely useful if you'd like to include custom fields to the package.
-
+This package includes all source columns defined in the macros folder. If you want to include custom fields in this package, you can add more columns using our pass-through column variables.
```

Comment:
Please double-check this content - I reworded for clarity, but I want to be sure I didn't change your original meaning

Reply:
original meaning persists and is much clearer. Thank you!